examples/simple-proposals.yaml

```diff
@@ -0,0 +1,262 @@
+    flowQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 16
+    hasBeginning: 2019-04-12T08:00:00-5:00
+    hasEnd: 2019-04-13T17:00:00-5:00
+
+  - '@id': fablab:33e8933b-ff73-4a01-964a-ca7a98893083
+    '@type': Proposal
+    name: Electrochemical engineering skills
+    hasBeginning: 2019-04-01T08:00:00-5:00
+    hasEnd: 2019-04-11T17:00:00-5:00
+    eligibleLocation: map:566a2c01-96f8-4b3c-adcf-f197eced3092 # Madison, WI
+
+  - '@id': fablab:b90b0b77-09a2-42e2-8bd4-e9ae2c1c6172
+    '@type': ProposedIntent
+    publishedIn: fablab:33e8933b-ff73-4a01-964a-ca7a98893083 # the proposal
+    publishes: fablab:60f4204e-b8d2-4026-8577-102c3f82c0af # the intent
+    reciprocal: false
+
+  - '@id': fablab:583e83d9-a46d-44ff-bd71-88513a1d83c0
+    '@type': Intent
+    name: Payment
+    skos:note: Payment when the unit is manufactured and sold
+    action: transfer
+    provider: https://fablab.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q4917 # US Dollar
+    under: https://fablab.example/soil-sensor-project/value-equation/
+
+  - '@id': fablab:d4d2fd71-34f2-41c3-b1c5-19ad5ed2da59b
+    '@type': ProposedIntent
+    publishedIn: fablab:33e8933b-ff73-4a01-964a-ca7a98893083 # the proposal
+    publishes: fablab:583e83d9-a46d-44ff-bd71-88513a1d83c0 # the intent
+    reciprocal: true
+
+  # Intent as part of 2 proposals
+
+  - '@id': alice:8e5fe80d-a769-4bd5-89e5-2136d33eab9f
+    '@type': Proposal
+    name: Free used bike # alice did not get any responses to her proposal above, so decided to offer the bike for free
+    hasBeginning: 2019-04-01T00:00:00-5:00
+    hasEnd: 2019-05-01T00:00:00-5:00
+    eligibleLocation: map:566a2c01-96f8-4b3c-adcf-f197eced3092 # Madison, WI
+
+  - '@id': alice:a8236bbb-81e0-422d-9861-56d2417db0fb
+    '@type': ProposedIntent
+    publishedIn: alice:8e5fe80d-a769-4bd5-89e5-2136d33eab9f # the proposal
+    publishes: alice:02b39a30-3e04-4305-9656-7f261aa63c84 # the intent (this is the same intent alice originally published)
+
+  # Price list
+
+  - '@id': farm:da7bfa38-a1b2-4f6f-a896-b9bf1ec8807a
+    '@type': Proposal
+    hasBeginning: 2019-03-01T00:00:00-5:00
+    hasEnd: 2019-03-31T00:00:00-5:00
+    eligibleLocation: map:566a2c01-96f8-4b3c-adcf-f197eced3092 # Madison, WI
+
+  - '@id': farm:c7897c39-7f05-4a5d-a487-80e130a24345
+    '@type': Intent
+    name: Carrots case 25#
+    action: transfer
+    provider: https://farm.example/
+    resourceClassifiedAs: wd:Q81 # carrot
+    resourceSpecifiedAs: farm:2402b57c-1c45-423a-acb2-f50961c97d65 # 25 pound case organic carrots seconds
+    unitQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': farm:a8236bbb-81e0-422d-9861-56d2417db0fb
+    '@type': ProposedIntent
+    publishedIn: farm:da7bfa38-a1b2-4f6f-a896-b9bf1ec8807a # the proposal
+    publishes: farm:c7897c39-7f05-4a5d-a487-80e130a24345 # the intent
+    reciprocal: false
```

Comment:
This doesn't provide any information on how to calculate USD to case of carrots exchange rate. It would need to have something like 

reciprocityQuantity:
qudt:unit: unit:Number
qudt:numericValue: 1

Reply:
1 what?